wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -799,7 +799,7 @@ private Subject sessionUser() {
 
     private String sessionUserNameOr(String nameIfNoUser) {
         final Subject subject;
-        if (EnvironmentContext.getCurrent() != null && (subject = EnvironmentContext.getCurrent().getSubject()) != null) {
+        if ((subject = EnvironmentContext.getCurrent().getSubject()) != Subject.ANONYMOUS) {
```

Comment:
@benoitf So this method is used only for logging. I don't know any way to test it and I think we can miss it(Mb you know?). Also we could make this method package private and test only its functionality, but I don't like this way in this case. Because it will be more testing of `EnvironmentContext#getSubject` method than `sessionUserNameOr` functionality :) 
Mb the best way is adding isAnonymous() method to Subject and don't use subject comparing in such use cases.